node/modules/builtin_actors.go

```diff
@@ -65,10 +66,25 @@ func LoadBuiltinActors(lc fx.Lifecycle, mctx helpers.MetricsCtx, r repo.LockedRe
 			return result, xerrors.Errorf("error loading %s from datastore: %w", key, err)
 		}
 
-		// ok, we don't have it -- fetch it and add it to the blockstore
-		mfCid, err := bundle.FetchAndLoadBundle(ctx, r.Path(), bs, av, rel, netw)
-		if err != nil {
-			return result, err
+		// we haven't recorded it in the daatastore, so we need to load it
+		var mfCid cid.Cid
+		if bd.Path != "" {
+			// this is a local bundle, load it directly from the filessystem
+			mfCid, err = bundle.LoadBundle(ctx, bs, bd.Path, av)
+			if err != nil {
+				return result, err
+			}
+		} else {
+			// fetch it and add it to the blockstore
+			mfCid, err = bundle.FetchAndLoadBundle(ctx, r.Path(), bs, av, bd.Release, netw)
+			if err != nil {
+				return result, err
+			}
+		}
```

Comment:
nit: Let's refactor this to a switch so that the different ways this can go is clear, and let's add a default branch that errors out when no source is specified:

```go
switch {
    case bd.Path != "":
        // use bundle from path
    case bd.Release != "":
        // use bundle from GitHub
    default:
        return nil, fmt.Errorf("no release or path specified for bundle for actor version %d", av)
}
```

Reply:
ok.